examples/category-results-materialize.html

```diff
@@ -132,7 +132,9 @@
           this.sourceFn = function (query) {
             if (query) {
               return populateSearch().then(function (fuse) {
-                const searchResults = fuse.search(query);
+                const searchResults = fuse.search(query).filter(function (result) {
+                  return !demo.model.includes(result);
```

Comment:
I went back and forth on it, but decided against it. Maybe an app maker would want to allow duplicates in their UI. Or they don't want it to disappear but merely appear disabled.